Formula/s/sbom-tool.rb

```diff
@@ -6,6 +6,11 @@ class SbomTool < Formula
   license "MIT"
   head "https://github.com/microsoft/sbom-tool.git", branch: "main"
 
+  livecheck do
```

Comment:
```suggestion
  # There can be a notable gap between when a version is tagged and a
  # corresponding release is created, so we check the "latest" release instead
  # of the Git tags.
  livecheck do
```

The `stable` URL is a tag archive (instead of a release asset), so we should document why `GithubLatest` is necessary here.